### PR TITLE
[netcore] Revise items in CoreFX.issues.rsp

### PR DESCRIPTION
**Pull request by EgorBo**

### CODE REVIEW

File: netcore/CoreFX.issues.rsp [hunk 1]
```diff
@@ -984,10 +923,6 @@
 ##  System.Threading.Overlapped.Tests
 ####################################################################
 
-# Overflow Exception
-# https://github.com/mono/mono/issues/15197
--nomethod OverlappedTests.PropertyTest2
```
Comment (filipnavara) [inline netcore/CoreFX.issues.rsp]: Still fails on non-Windows, as expected.

File: netcore/System.Private.CoreLib/src/System.Runtime.InteropServices/Marshal.cs [hunk 2]
```diff
@@ -409,6 +409,16 @@ public static void WriteInt64(Object ptr, int ofs, long val)
 			throw new PlatformNotSupportedException ();
 		}
 
+		public static int GetHRForException(Exception? e)
```
Comment (marek-safar) [inline netcore/System.Private.CoreLib/src/System.Runtime.InteropServices/Marshal.cs]: Why do you need this, the methods already exist

Comment (EgorBo) [reply netcore/System.Private.CoreLib/src/System.Runtime.InteropServices/Marshal.cs]: it exists in `Marshal.NoCom.cs` for unix. so ideally these methods should be moved to `Marshal.Windows.cs` but I'll rollback these changes - there is no sense in declaring `FeatureCominterop` and not implementing it

**Comment by steveisok** [comment]
@monojenkins build failed

**Comment by MaximLipnin** [comment]
Seems that https://github.com/mono/mono/issues/14913 can be closed now

**Pull request merged**